Wikipdedia/practice_beautifulsoap.py

```diff
@@ -0,0 +1,69 @@
+from bs4 import BeautifulSoup
+import requests
+
+language_symbols = {}
+
+
+def lang():
+    try:
+        response = requests.get("https://www.wikipedia.org/")
+        response.raise_for_status()
+        soup = BeautifulSoup(response.content, 'html.parser')
+
+        for option in soup.find_all('option'):
+            language = option.text
+            symbol = option['lang']
+            language_symbols[language] = symbol
```

Comment:
Also, provide a proper README.md file, for other people, general people to understand what is this project is all about.
and cover basic aspects like:
- How to install
- How to uninstall
- How to use
etc...,
What are the key features, what are bugs, what to improve, what can be further refactored, etc..., just explaining your project in full length.